corehq/apps/data_analytics/migrations/0002_maltrow_threshold.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('data_analytics', '0001_squashed_0004_auto_20150810_1710'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='maltrow',
+            name='threshold',
+            field=models.PositiveSmallIntegerField(default=15),
+            preserve_default=True,
```

Comment:
The issue is that it could make the deploy take a long time due to the SQL
migration but I don't think 60K rows should be too bad.

On 23 November 2015 at 17:18, Cal Ellowitz <notifications@github.com> wrote:

> In corehq/apps/data_analytics/migrations/0002_maltrow_threshold.py
> <https://github.com/dimagi/commcare-hq/pull/9218#discussion_r45614499>:
>
> > +
> > +from django.db import models, migrations
> > +
> > +
> > +class Migration(migrations.Migration):
> > +
> > +    dependencies = [
> > +        ('data_analytics', '0001_squashed_0004_auto_20150810_1710'),
> > +    ]
> > +
> > +    operations = [
> > +        migrations.AddField(
> > +            model_name='maltrow',
> > +            name='threshold',
> > +            field=models.PositiveSmallIntegerField(default=15),
> > +            preserve_default=True,
>
> I looked into the maltrow table and it is about 60k rows right now. I dont
> really know how table rewrites work in postgres, but the malt table is
> updated only once per month (on the second), so even a large operation on
> that table shouldn't block any functionality unless it will have negative
> effects on our other postgres work
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/9218/files#r45614499>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa